woocommerce/woocommerce
# Remove the product block breadcrumbs and sidebar inspector

### All Submissions:






### Changes proposed in this Pull Request:

Removes the unused block breadcrumbs and sidebar inspector from the new product blocks experience per the Figma designs.

<img width="929" alt="Screen Shot 2023-03-15 at 11 18 41 AM" src="https://user-images.githubusercontent.com/10561050/225405644-f25d8400-9dc4-4720-9b68-961e3cb96d43.png">


Closes #37244  Closes #37245



### How to test the changes in this Pull Request:




1. Enable the new product blocks experience
2. Navigate to the add new product page
3. Verify that the blocks breadcrumbs (beneath the header and tabs) and the sidebar blocks inspector no longer appear



### Other information:




### FOR PR REVIEWER ONLY:
